### PR TITLE
Tilt improvements: Add visualizer, better defaults

### PR DESCRIPTION
**Pull request by hrydgard**
Helps #16881 

Turns out it was nearly impossible to understand how tilt controls worked without a visualizer, so I added one.

And it was revealed that my understanding was wrong too (I didn't write the code originally), especially around calibration.

I'll followup with a change that will make it work better held at an angle. This will remove the Base Position setting and replace with just calibration.

Anyway, here's the first improvement to the UI, which already makes it much easier to configure:

image

### CODE REVIEW

**Comment by iota97** [comment]
There might be some minor issues with the translations, for example in Italian one:

```
Calibrate = Calibra il D-Pad
To Calibrate = Mantenere il dispositivo su una superficie piatta e premere "Calibra".
```

"To Calibrate" will ask to press "Calibra", but the button will actually be called "Calibra il D-Pad". Same with few other languages. Probably it's quite confusing it's referencing D-Pad here at all :/

File: assets/lang/ja_JP.ini [hunk 1]
```diff
@@ -35,7 +35,7 @@ Button Binding = 対応ボタンの割り当て（複数同時設定可能）
 Button Opacity = ボタンの透明度
 Button style = ボタンのスタイル
 Calibrate Analog Stick = アナログスティックをキャリブレートする
-Calibrate D-Pad = 十字キーをキャリブレートする
+Calibrate = 十字キーをキャリブレートする
```
Comment (Narugakuruga) [inline assets/lang/ja_JP.ini:38]: キャリブレート

File: assets/lang/lt-LT.ini [hunk 2]
```diff
@@ -35,7 +35,7 @@ Button Binding = Button Binding
 Button Opacity = Mygtukų opacity
 Button style = Mygtukų stilius
 Calibrate Analog Stick = Calibrate Analog Stick
-Calibrate D-Pad = Kalibruoti krypčių mygtukus
+Calibrate = Kalibruoti krypčių mygtukus
```
Comment (Narugakuruga) [inline assets/lang/lt-LT.ini:38]: Kalibruoti

File: assets/lang/tr_TR.ini [hunk 3]
```diff
@@ -35,7 +35,7 @@ Button Binding = Düğme Bağlama
 Button Opacity = Tuş görünürlüğü
 Button style = Tuş stili
 Calibrate Analog Stick = Analog Çubuğu Kalibre Et
-Calibrate D-Pad = D-Pad'i kalibre et
+Calibrate = D-Pad'i kalibre et
```
Comment (Narugakuruga) [inline assets/lang/tr_TR.ini:38]: kalibre

File: assets/lang/ca_ES.ini [hunk 4]
```diff
@@ -35,7 +35,7 @@ Button Binding = Assignat de botons
 Button Opacity = Transparència dels botons
 Button style = Estil dels botons
 Calibrate Analog Stick = Calibrate Analog Stick
-Calibrate D-Pad = Calibrar la creueta
+Calibrate = Calibrar la creueta
```
Comment (Narugakuruga) [inline assets/lang/ca_ES.ini:38]: Calibrar

**Pull request merged**